src/types.ts

```diff
@@ -16,6 +16,7 @@ export interface Account {
   connector: ConnectorType;
   config: string;
   name: string;
+  nostrPrivateKey: string | null;
```

Comment:
can this be optional? `nostrPrivateKey?` ?

Reply:
Yes, changing it!